### PR TITLE
Add test script to make it easier to run the tests

### PR DESCRIPTION
**Pull request by carusogabriel**
Everytime that I need to run our cs's tests, I need to remember this command. Let's add it to composer's scripts and expose it with a simple `composer test` :)

### CODE REVIEW

**Comment by Majkl578** [comment]
I don't like using Composer for executing scripts - I'd prefer a simple bash script instead, so 👎 from me.

**Comment by greg0ire** [comment]
Where do you usually put that kind of script? `bin`? `scripts`? Something else?

**Comment by carusogabriel** [comment]
. @greg0ire has a point. I got that we won't use the composer `scripts` way, but I'd like to have it somewhere to make easier to run the tests

**Comment by Ocramius** [comment]
Just `bin` - works for most projects in the wild

**Comment by Majkl578** [comment]
`bin/` is fine, in case we start exporting `bin/` by composer, we can export-ignore dev-only scripts. :)

**Comment by ostrolucky** [comment]
https://localheinz.com/blog/2018/01/24/makefile-for-lazy-developers/

**Comment by Majkl578** [comment]
Eww, no makefiles please. 😭

**Comment by greg0ire** [comment]
@ostrolucky I like using `make` too, but I always think in the back of my mind that when everything is a `.PHONY`, then I am probably misusing it :P If the point is wrapping, a simple shell script might be better indeed.

@carusogabriel please move this to a bash script inside `bin` then, since everyone seems to be fine with it.

**Comment by ostrolucky** [comment]
I think you don't understand advantage of build system in comparison with bash scripts. It is aware of dependent tasks and runs them only when needed. Look at the starting point in linked article. It's super simple and exactly what is needed here, you can't make it simpler in bash.

**Comment by greg0ire** [comment]
There is also parallelization (`--jobs`) and `--output-size`, my point is, it is only good if we really use it, and we would only use it for `composer`, right? The goal of this PR is neither leveraging the dependency system or parallelization, but having a shortcut for a series of shell commands. To be able to take a decision, we should have a good list of arguments against `make`, so, here's a start, maybe @Majkl578 @Ocramius have something to add (besides eww :trollface: ) …

- it's one more dependency, whereas shell (bash?) is everywhere (is it though? hi docker!)
- the command lines use make syntax, and produce command lines, and will be syntactically colored as make, not as shell

File: bin/test.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+./vendor/bin/phpcs tests/input --report=summary --report-file=phpcs.log; diff tests/expected_report.txt phpcs.log
```
Comment (greg0ire) [inline bin/test.sh]: Please use linebreaks

**Pull request merged**